fenix/app/src/main/res/values/preference_keys.xml

```diff
@@ -42,10 +41,8 @@
     <string name="pref_key_override_amo_user" translatable="false">pref_key_override_amo_user</string>
     <string name="pref_key_override_amo_collection" translatable="false">pref_key_override_amo_collection</string>
     <string name="pref_key_enable_gecko_logs" translatable="false">pref_key_enable_gecko_logs</string>
-    <string name="pref_key_help" translatable="false">pref_key_help</string>
```

Comment:
aren't you cleaning these up in https://github.com/mozilla-mobile/firefox-android/pull/5320?

Reply:
@MozillaNoah yep! This PR depends on that issue, which is why it's included as a commit here, as I wanted to keep the 2 refactors separate 🙂

Reply:
I really need to have coffee before doing your PRs 😂